plugin.json

```diff
@@ -7,11 +7,13 @@
   "release_notes_url": "https://github.com/mattermost/mattermost-plugin-autolink/releases/tag/v1.2.1",
   "icon_path": "assets/icon.svg",
   "version": "1.2.1",
-  "min_server_version": "5.16.0",
+  "min_server_version": "5.37.0",
```

Comment:
@hanzei Is this actually correct, or should we require 6.0?

Reply:
following the same approach for the other PRs that Ben suggest to update this. i can make a change to what fits best

Reply:
5.37.0 is correct.